tests/core/test_lane.py

```diff
@@ -4094,13 +4094,13 @@ def _w(**kwargs):
 
         # Create children works.
         result.children_with_age = _w(
-            title="Children work with set target age",
+            title="Children work with target age",
             audience=Classifier.AUDIENCE_CHILDREN,
         )
-        result.children_with_age.target_age = (0, 3)
+        result.children_with_age.target_age = tuple_to_numericrange((0, 3))
 
         result.children_without_age = _w(
-            title="Children work without target age",
+            title="Children work with out target age",
```

Comment:
Same as above, making sure the sorting happens in the same way with both back-ends